src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -25,7 +25,7 @@ const RecentSearchItems = (prop) => {
         borderless: false
       }}>
       <Text style={styles.text}>{text}</Text>
-      <IcArrowUpRight style={styles.icon} fill={COLORS.white2} />
+      <IcArrowUpRight style={styles.icon} fill={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `IcArrowUpRight` icon has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change is intentional and that it doesn't affect the visibility of the icon against its background.
```diff
- <IcArrowUpRight style={styles.icon} fill={COLORS.white2} />
+ <IcArrowUpRight style={styles.icon} fill={COLORS.white} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->